llvm/include/llvm/Object/COFFImportFile.h

```diff
@@ -137,6 +137,7 @@ struct COFFShortExport {
 
 Error writeImportLibrary(StringRef ImportName, StringRef Path,
                          ArrayRef<COFFShortExport> Exports,
+                         ArrayRef<COFFShortExport> NativeExports,
```

Comment:
For users of this API, could you add a comment about the difference between Exports and NativeExports?

(Rust is pondering what to do here: https://github.com/rust-lang/rust/pull/120996)

Reply:
Sure, I created #81600 with a clarification.